packages/dns/package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@google-cloud/common": "^0.5.0",
     "arrify": "^1.0.0",
-    "dns-zonefile": "0.1.18",
+    "dns-zonefile": "^0.1.18",
```

Comment:
@callmehiphop We intentionally wanted 0.1.18 and not any updates, didn't we? I can exclude this if so.